model/src/main/java/org/projectnessie/api/v2/doc/ApiDoc.java

```diff
@@ -84,19 +93,14 @@ public interface ApiDoc {
           + FULL_REF_INFO;
 
   String CHECKED_REF_DESCRIPTION =
-      "Specifies a named branch or tag reference.\n"
+      "Specifies a named branch or tag reference with its expected HEAD 'hash' value.\n"
           + "\n"
-          + "A named reference can be specification in these forms:\n"
-          + "- name - Identifies a branch or tag without a concrete HEAD 'hash' value.\n"
+          + "For example:\n"
           + "- name@hash - Identifies the 'hash' commit on a branch or tag.\n"
           + "\n"
-          + "If both 'name' and 'hash' are given, 'hash' must be the current HEAD of the branch or tag. It will be "
-          + "used to validate that at execution time the reference points to the same hash that the caller expected "
-          + "when the operation arguments were constructed.\n"
-          + "\n"
-          + "Not specifying the 'hash' value relaxes server-side checks and can lead to unexpected side effects if "
-          + "multiple changes to the same reference are executed concurrently. It is recommended to always specify the "
-          + "'hash' value when assigning or deleting a reference.\n";
+          + "The specified 'hash' must be the current HEAD of the branch or tag. It will be used to validate that "
```

Comment:
```suggestion
          + "The specified 'hash' must be the value of the current HEAD of the branch or tag known by the client. It will be used to validate that "
```